src/pages/SkillSharePage/components/Steps/Skills/Skills.tsx

```diff
@@ -0,0 +1,83 @@
+import Button from "@/src/components/SeButton/SeButton";
+import Tag from "@/src/components/Tag/Tag";
+import { FormikProps, FormikValues } from "formik";
+import { useState } from "react";
+
+type SkillsStepProps = {
+  form: FormikProps<FormikValues>;
+  name: "Skills";
+};
+
+function SkillsStep({ form }: SkillsStepProps) {
+  const [formSkills, setFormSkills] = useState<string[]>([])
+  const [skillItem, setSkillItem] = useState('');  
+
+  form.values.skills = formSkills
+
+  const addFormSkill = () => {
+    if (skillItem && !formSkills.includes(skillItem)) {
+      setFormSkills([...formSkills, skillItem]);
```

Comment:
You are passing in form from the Formik provider and you can begin to make use of it. One of the key things about Formik is updating with setFieldValue instead of local useStates.

```
const addFormSkill = () => {
  if (skillItem && !form.values.skills.includes(skillItem)) {
    const updatedSkills = [...form.values.skills, skillItem];
    form.setFieldValue("skills", updatedSkills);
    setSkillItem("");  // or skillItemInputField
  }
}
```

Reply:
You would remove skill items the same way, filtering out the form.values.skills array for the skill you want to remove and then setting the new value.

Reply:
This worked. Thank you.